### ISSUE TITLE
Upon not receiving any input for the tldr command, it should invoke itself 

### ISSUE THREAD

**Issue by Parth-Mehrotra** [tldr-pages/tldr-cpp-client#8]
if the user types in just `tldr` only to have no output generated. We could output something like this: 
```
Usage: tldr [OPTIONS] COMMAND [ARGS]...

  A python client for tldr: simplified and community-driven
  man pages.

Options:
  -V, --version  Show the version and exit.
  -h, --help     Show this message and exit.

Commands:
  find    Find the command usage.
  init    Init config file.
  update  Update to the latest pages.
```

This is what the python client does, or we could simply invoke `tldr tldr`

**Comment by Parth-Mehrotra** [comment]
(would be happy to contribute a PR, if this seems useful to you guys)

**Comment by Leandros** [comment]
I don't like to invoke `tldr tldr`, but yes, it's missing a help message and should definitely show one. 
Would love to see you'r pull request.

**Comment by Parth-Mehrotra** [comment]
Pull request: #9

**Comment by Leandros** [comment]
Closed in #9.
